File: forge/src/main/java/net/spaceeye/vmod_additions/forge/VModAdditionsForge.java
package net.spaceeye.vmod_additions.forge;

import dev.architectury.platform.forge.EventBuses;
import net.spaceeye.vmod_additions.VModAdditions;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;

@Mod(VModAdditions.MOD_ID)
public class VModAdditionsForge {
    public VModAdditionsForge() {
		// Submit our event bus to let architectury register our content on the right time
        EventBuses.registerModEventBus(VModAdditions.MOD_ID, FMLJavaModLoadingContext.get().getModEventBus());
        VModAdditions.init();
    }
}

File: forge/src/main/kotlin/net/spaceeye/vmod_additions/forge/VModAdditionsForge.kt
package net.spaceeye.vmod_additions.forge

import dev.architectury.platform.forge.EventBuses
import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext
import net.spaceeye.vmod_additions.VA
import net.spaceeye.vmod_additions.VA.init

@Mod(VA.MOD_ID)
class VModAdditionsForge {
    init {
        // Submit our event bus to let architectury register our content on the right time
        EventBuses.registerModEventBus(VA.MOD_ID, FMLJavaModLoadingContext.get().modEventBus)
        init()
    }
}